### ISSUE TITLE
bug: Gaiad v7.1.0 cannot run in WSL2 of Windows 11 (21H2 version)

### ISSUE THREAD

**Issue by ashuralyk** [cosmos/gaia#1833]
## Summary of Bug
Just have no reaction after running `gaiad version` and have no logs print yet. I also have tried to build Gaiad from Go source files, but aslo got the same result, just no reaction. In addition, `$HOME/.gaia` directory was created.
![image](https://user-images.githubusercontent.com/3871265/196628515-d141f697-8af5-4eec-af4a-b3b1d4cf0d73.png)

BTW, `tendermint version` (github.com/tendermint/tendermint.git) works fine.
![image](https://user-images.githubusercontent.com/3871265/196629869-0a633f33-67a4-4ce5-be8e-87a99ada00d8.png)

## Steps to Reproduce

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned
- [ ] Is a spike necessary to map out how the issue should be approached?

**Comment by Pantani** [comment]
Hey @ashuralyk, thanks for opening the issue. What happens if you type only `gaiad` or `gaiad -h`? You can see usage help.

**Comment by ashuralyk** [comment]
> Hey @ashuralyk, thanks for opening the issue. What happens if you type only `gaiad` or `gaiad -h`? You can see usage help.

nothing happened in the screen when I type `gaiad`, just have NOTHING in reaction, it's looking like the binary file has broken down which I got by compiling from source file and downloading the precompiled file.

![image](https://user-images.githubusercontent.com/3871265/197820951-d9822924-67fc-4c22-81f7-7756764c7847.png)

**Comment by Pantani** [comment]
Can you try to download the binary e run it in the local folder?
e.g.: Download the [latest windows binary](https://github.com/cosmos/gaia/releases/tag/v7.1.0) and run `./gaiad-v7.1.0-windows-amd64.exe` in the same folder you downloaded

**Comment by ashuralyk** [comment]
> Can you try to download the binary e run it in the local folder? e.g.: Download the [latest windows binary](https://github.com/cosmos/gaia/releases/tag/v7.1.0) and run `./gaiad-v7.1.0-windows-amd64.exe` in the same folder you downloaded

Yep, it works fine in Windows 11 Command line.

![image](https://user-images.githubusercontent.com/3871265/197957450-369a90e2-2b2e-41d0-9b21-054902b9b033.png)

**Comment by Pantani** [comment]
Your binary seems compromised. Try to replace it with the new one you already downloaded.

**Comment by ashuralyk** [comment]
> Your binary seems compromised. Try to replace it with the new one you already downloaded.

I'm sorry, the binary I previously downloaded cannot run in WSL2 command line, because WSL2 a simulated Linux CMD running in Windows 11, but the binary is compiled just for usage in Windows 11, so the problem is here, maybe there is some misunderstanding between us.

**Comment by Pantani** [comment]
Oh, I see. Sorry, I don't know how the WSL2 works. How did you get the WSL2 binary? Did you try to build from the source or download it?

**Comment by ashuralyk** [comment]
> Oh, I see. Sorry, I don't know how the WSL2 works. How did you get the WSL2 binary? Did you try to build from the source or download it?

Yes, I tried both two ways, but all of them resulted in the same result which was no reaction in the WSL2 command line, so I got no idea what to do next.

**Comment by mmulji-ic** [comment]
Hi @ashuralyk ,

I tried to replicate the issue on my Mac M1, running Windows 11 using Parallels. I used the following [link](https://github.com/cosmos/gaia/releases/tag/v7.1.0).

I got the windows binary to run, which displayed the expected output as shown here:

<img width="924" alt="image" src="https://user-images.githubusercontent.com/98309852/199490132-6cf57b19-166a-4867-bdcd-4aea3cbc6fd1.png">

I tried the amd image, that did not work, but the arm based one generated the expected output:

<img width="658" alt="image" src="https://user-images.githubusercontent.com/98309852/199491179-5147e74e-654f-49cc-9b11-90426d4aeb84.png">

However, this is not a usecase we think is common and one that we should support. There are lots of potential OS / setup issues that could occur. We would advise a more vanilla setup.
